### ISSUE TITLE
Move normalization and error clipping to parameter server side.

### ISSUE THREAD

**Issue by typhoonzero** [PaddlePaddle/Paddle#7432]
Move normalization and error clipping to parameter server side in the distributed transpiler.

**Comment by helinwang** [comment]
- I think it the normalization is done on the trainer side, making it part of the loss function, then the gradient update (parameter server's job) is element wise, so we don't need to do anything special. I briefly looked into the code few days ago, it seems that the normalization is already part of the loss function.

- Gradient clipping we can do on the parameter server side, it seems not very hard.

**Comment by typhoonzero** [comment]
Sorry, my fault, I mean regularization and clipping. See: https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/optimizer.py#L203

Thanks very much for your comment.

**Comment by helinwang** [comment]
Thanks, very important information, I will consider this.

**Comment by typhoonzero** [comment]
I think the simple version could put decay and clipping to server side should do the basic work.
